src/interactionManager/interactionManager.ts

```diff
@@ -16,6 +16,7 @@ flows.forEach((f) => {
 export interface InteractionEvents {
```

Comment:
this can be abstracted into this:
```
type InteractionEventKeys = "interactionCreated" | "interactionUpdated" | "interactionResumed"
export type InteractionEvents = {
  [key in InteractionEventKeys]: (interxn: Interaction) => void
}